src/ceph-disk/ceph_disk/main.py

```diff
@@ -1922,6 +1922,10 @@ def parser():
             help='unique OSD uuid to assign this disk to',
         )
         parser.add_argument(
+            '--crush-class',
```

Comment:
maybe we should change **osd crush set-device-class** to **osd crush set-crush-class** so they all look the same ?